src/routes/paths.svelte

```diff
@@ -77,7 +77,14 @@
 
 <Spacing amount="175px" />
 
-<WhatAreYouWaitingFor />
+<CallToActionSection>
+  <h2 slot="headline">how can i get involved?</h2>
+  <p>

```

Comment:
Currently, the `/paths` route is missing the paragraph element provided by `<CallToActionSection>`. I believe the code below can help.

```html
<p slot="text">
````